migrate-mongo-store.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * Crowi::migrateStore
+ *
+ * @package Crowi
+ * @author  otofune <otofune@gmail.com>
+ */
+
+const mongoose = require('mongoose')
+
+const crowi = new (require('./lib/crowi'))(__dirname, process.env)
+
+const migrationSchema = new mongoose.Schema({
+  title: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  at: {
+    type: Date,
+    required: true,
+    index: true,
+  },
+  status: {
+    type: String,
+    required: true,
+    enum: ['up', 'down'],
+  },
+  history: [
+    {
+      status: {
+        type: String,
+        required: true,
+        enum: ['up', 'down'],
+      },
+      at: {
+        type: Date,
+        required: true,
+      },
+    },
+  ],
+})
+migrationSchema.methods.toEntry = function() {
+  const m = this
+  const entry = {
+    title: m.title,
+    timestamp: m.status === 'up' ? m.at.getTime() : null,
+  }
+  return entry
```

Comment:
literal variable `entry` is not accessed as left value. So the declaration is not needed.

just
```
return {
    title: this.title,
    timestamp: this.status === 'up' ? this.at.getTime() : null,
  }
```